Sources/TuistKit/Commands/Cache/CacheOptions.swift

```diff
@@ -20,4 +21,12 @@ struct CacheOptions: ParsableArguments {
         help: "When passed it caches the targets for simulator and device using xcframeworks."
     )
     var xcframeworks: Bool = false
+
+    @Option(
+        name: .long,
+        help: "Output type of xcframeworks when --xcframeworks is passed (device/simulator)"
+    )
+    var destination: CacheXCFrameworkDestination?
```

Comment:
if we make this an OptionSet, the default should be `[.device, .simultator]` (not sure how Option sets play with swift parser)